sdrf_pipelines/openms/openms.py

```diff
@@ -134,7 +134,7 @@ def openms_convert(self, sdrf_file: str = None, keep_raw: bool = False, one_tabl
         if not split_by_columns:
             # get factor columns (except constant ones)
             factor_cols = [c for ind, c in enumerate(sdrf) if
-                           c.startswith('factor value[') and len(sdrf[c].unique()) > 1]
+                           c.startswith('factor value[') and len(sdrf[c].unique()) >= 1]
 
             # get characteristics columns (except constant ones)
```

Comment:
What about "characteristics"? Do you want to do the same here?

Reply:
Yes it should be the same!!!